image_cropping/widgets.py

```diff
@@ -64,7 +64,7 @@ class CropWidget:
     def _media(self):
         js = [
             "image_cropping/js/jquery.Jcrop.min.js",
```

Comment:
This line should also be obsolete, as due to the concatenation this file is now part of `image_cropping.min.js`